scaling.adoc

```diff
@@ -243,92 +243,92 @@ different versions of Bitcoin Core going back to version 0.8.
 image::libsecp256k1speedups.png[{half-width}]
 
-The graph shows the trend for two different 64-bit CPU types, ARM and x86.
+The graph shows the trend for two different 64-bit CPU types, namely ARM and x86.
 The difference in performance is due to the more specialized instructions
-available on x86 compared to the ARM architecture, which has fewer,
-more generic instructions. But the general trend is the same for both
+available on x86 compared to the ARM architecture, which has fewer and
+more generic instructions. However, the general trend is the same for both
 architectures. Note that the Y-axis is logarithmic, which makes the
 improvements look less impressive than they actually are.
 
-There are also several good examples of space savings contributing to
-performance improvements. In a
+There are also several good examples of space-saving improvements that contributed to
+performance enhancement. In a
 https://murchandamus.medium.com/2-of-3-multisig-inputs-using-pay-to-taproot-d5faf2312ba3[Medium
-blog post] about Taproot's contribution to space savings, user Murch
-compared how much block space a 2-of-3 threshold signature would
-require, both without using Taproot and using Taproot in various ways.
+blog post] about Taproot's contribution to saving space, user Murch
+compares how much block space a 2-of-3 threshold signature would
+require, using Taproot in various ways as well as not using it at all.
 
-.Space savings for different spending types Taproot and legacy versions.
+.Space savings for different spending types, Taproot and Legacy versions.
 image::murch-taproot.png[{half-width}]
 
-A 2-of-3 multisig using native segwit would require a total of
-104.5+43 vB = 147.5 vB, while the most space conservative Taproot
-usage would in the standard use case require only 57.5+43 vB = 100.5
-vB. At worst, in rare cases, like when a standard signer is
-not available for some reason, 107.5+43 vB = 150.5 vB. You don't have
-to understand all the details, but it should give you an idea
-of how developers think about space savings. Every little byte counts.
-
-Apart from the inward scaling going on in Bitcoin software, there are
-also some ways that users can contribute to inward scaling. They can
-make their transactions in more intelligent ways to save on
+A 2-of-3 multisig using native Segwit would require a total of
+104.5+43 vB = 147.5 vB, whereas the most space-conservative use of Taproot
+would require only 57.5+43 vB = 100.5
+vB in the standard use case. At worst and in rare cases, like when a standard signer is
+not available for some reason, Taproot would use 107.5+43 vB = 150.5 vB. You don't have
+to understand all the details, but this should give you an idea
+of how developers think about saving space - every little byte counts.
+
+Apart from inward scaling in Bitcoin software, there are
+some ways in which users can contribute to inward scaling, too. One is that they can
```

Comment:
"One is that they can" sounds like there are more ways to do it, but I think there's not.

"there are some ways in which users can contribute to inward scaling" refers to the two techiques in the next sentence.

Reply:
Ok, you're right, now I see where the confusion came from. I'll change it back.